packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -1045,10 +1049,12 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
             switch (summaryObject.type) {
                 case api.SummaryType.Tree: {
+                    treesPathToSummaryTreeCacheLatest.set(`.app${path}/${key}`, summaryObject);
```

Comment:
minor, but it would be great to have a / after .app (I assume that requires changes in other places as well to stay consistent across all places where we use treesPathToSummaryTreeCacheLatest).
Or maybe we can drop .app part and just make sure that the only blobs we ever look are blobs under .app sub-tree? That's true for upload, but likely not true for download of snapshot (where we calculate shas for all blobs)?

Reply:
There is "/" after ".app" part. path in `.app${path}/${key}` actually start with a "/".